app/policies/tournament_policy.rb

```diff
@@ -4,5 +4,9 @@ class Scope < Scope
     def resolve
       scope.all
     end
+
+    def index
```

Comment:
Was this needed after all? I thought it worked with just the policy_scope thing you had in the tournament controller

Reply:
I was told yes... but not positive.